src/index.js

```diff
@@ -1,12 +1,31 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
+const list = document.querySelector('.list');
+const toDos = [
+  {
+    task: 'Join morning session meeting',
+    done: false,
+    index: 0,
+  },
+  {
+    task: 'Join my coding partners',
+    done: false,
+    index: 1,
+  },
+  {
+    task: 'Have a lunch break',
+    done: false,
+    index: 2,
+  },
+];
```

Comment:
- [ ] on load our list is great rendered to the DOM :clap:  But from the project requirement you need to render them in the order of their index. When I changed the index value the order doesn't change :cry:  Please fix it in order to make your project work as expected :smile: 

I think this piece of code can help you fix it :arrow_right:   **Array.sort((a, b) => a.index - b.index**